app/models/oauth_application.rb

```diff
@@ -4,4 +4,5 @@
 class OauthApplication < ApplicationRecord
   include ::Doorkeeper::Orm::ActiveRecord::Mixins::Application
   has_many :jwt_identifiers, foreign_key: 'application_id', dependent: :destroy
+  belongs_to :jurisdiction, optional: true
```

Comment:
I did have that in their initially here, but I removed it with a [this commit](https://github.com/SaraAlert/SaraAlert/pull/494/commits/ffc64e8bc108a2cb7e7b118ca2bde9a6024f9990), because we were no longer using it anywhere. For some reason even though it was `optional` it broke the user workflow tests if I didn't specify it.

Reply:
That's interesting, be good to understand that one better. Not necessary to get in, but odd that it would cause it issues and would be nice to get in for consistency.